benchmarks/Cargo.toml

```diff
@@ -4,11 +4,19 @@ version = "0.1.0"
 authors = ["benjirewis <ben.rewis98@gmail.com>"]
 edition = "2018"
 
+[features]
+default = ["tokio-runtime"]
+tokio-runtime = ["tokio/macros", "tokio/rt-core", "tokio/rt-threaded"]
+async-std-runtime = ["async-std"]
+
 [dependencies]
 mongodb = { path = ".." }
-bson = "0.13.0"
-serde_json = "1.0.39"
-lazy_static = "1.3.0"
-num_cpus = "1.10.1"
-clap = "2.33.0"
-indicatif = "0.11.0"
+serde_json = "1.0.59"
```

Comment:
I upgraded all of the existing dependencies to their latest versions without running into any issues